### DISCUSSION TITLE
"Uncaught Error: Config file /config/runtime.json cannot be read." in console

### DISCUSSION THREAD

**Discussion by unistar42** [community/community#112069] [Programming Help]
### Select Topic Area

Bug

### Body

I am developing a project with laravel-backend and react-frontend. There are no other problems with the backend, and when I run the front, nothing is displayed on localhost:3000, and the following error appears in the console window.

`config.js:805 Uncaught Error: Config file /config/runtime.json cannot be read. Error code is: undefined. Error message is: FileSystem.readFileSync is not a function
    at push../node_modules/config/lib/config.js.util.parseFile (config.js:805:1)
    at push../node_modules/config/lib/config.js.util.loadFileConfigs (config.js:685:1)
    at new Config (config.js:116:1)
    at Object.<anonymous> (config.js:1442:1)
    at ./node_modules/config/lib/config.js (config.js:1453:1)
    at __webpack_require__ (bootstrap:856:1)
    at fn (bootstrap:150:1)
    at Module.<anonymous> (api.js:14:1)
    at ./src/@mock-api/data/apps.chat.js (apps.chat.js:453:1)
    at __webpack_require__ (bootstrap:856:1)
    at fn (bootstrap:150:1)
    at Module.<anonymous> (users.js:151:1)
    at ./src/@mock-api/index.js (index.js:13:1)
    at __webpack_require__ (bootstrap:856:1)
    at fn (bootstrap:150:1)
    at Module.<anonymous> (constants.js:49:1)
    at ./src/index.js (index.js:87:1)
    at __webpack_require__ (bootstrap:856:1)
    at fn (bootstrap:150:1)
    at 1 (store.js:44:1)
    at __webpack_require__ (bootstrap:856:1)
    at checkDeferredModules (bootstrap:45:1)
    at Array.webpackJsonpCallback [as push] (bootstrap:32:1)
    at main.chunk.js:1:65`

These errors appeared at the last step in fixing npm errors.

This is a webpack.config.js file.

`const path = require('path');
const HtmlWebpackPlugin = require('html-webpack-plugin');
const MiniCssExtractPlugin = require('mini-css-extract-plugin');
const webpack = require('webpack');

module.exports = {
  entry: './src/index.js', // Adjust the entry point according to your project structure
  output: {
    path: path.resolve(__dirname, 'dist'), // Output directory
    filename: 'bundle.js', // Output bundle file name
  },
  resolve: {
    modules: [
      'node_modules',
      path.resolve(__dirname, 'src'), // Add your source directory here if needed
    ],
    extensions: ['.js', '.jsx', '.json'], // Add any other extensions you might need
    fallback: {
      fs: false,
      path: require.resolve('path-browserify'),
    },
  },
  module: {
    rules: [
      {
        test: /\.js$/,
        exclude: /node_modules/,
        use: {
          loader: 'babel-loader',
        },
      },
      {
        test: /\.css$/,
        use: [MiniCssExtractPlugin.loader, 'css-loader'],
      },
    ],
  },
  plugins: [
    new webpack.DefinePlugin({
      'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV),
      'process.env.BROWSER': true
    }),
    new HtmlWebpackPlugin({
      template: './public/index.html', // Path to your HTML template
      filename: 'index.html',
    }),
    new MiniCssExtractPlugin({
      filename: '[name].css',
      chunkFilename: '[id].css',
    }),
  ],
  devServer: {
    contentBase: path.resolve(__dirname, 'dist'), // Serve content from the 'dist' directory
    port: 3000, // Choose your preferred port
    open: true, // Open the browser automatically when the server starts
    setupMiddlewares: function (devServer) {
      // Your middleware setup code here
    },
  },
  //   node: {
  //     fs: 'empty'
  //   }
};`

![console error](https://github.com/community/community/assets/151569262/04053055-0118-4667-ae09-52375ffb0ad9)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @danieleverest ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by alfredo-fnz007** [comment]
you have to do npm install in frontend again, and then try project.
